nginx/conf.d-test/default.conf

```diff
@@ -1,13 +1,31 @@
+resolver 127.0.0.11 valid=30s;
```

Comment:
Perhaps we should add a comment here that that is the Docker namespace resolver.
Same in the other file